notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.hist(true_x, bins=20)
</pre><p> </p><p>Not sure why this is not working in my system using the latest jax jaxlib seaborn matplotlib. My workaround is to use plt.plot(x.copy()) here and other plt.hist plots. :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Hi @fehiepsi , I couldn't reproduce this. What versions of jax/matplotlib do you have?

Reply:
It also happens in CI (e.g. [this run](https://github.com/fehiepsi/rethinking-numpyro/runs/4696416337?check_suite_focus=true) that installs `matplotlib-3.5.1` `jax-0.2.26` `jaxlib-0.1.75`) and I fixed that issue by using [.copy](https://github.com/fehiepsi/rethinking-numpyro/commit/8115e954f1bb3226110c7e95a09d72eea42546aa#diff-9cae103a75e9ba92f55f71040a13b4b661f9862b55cb9225a98422f357570f0cR146). I got the same issues locally in my system with your tutorial.

You can test your tutorial in colab by adding
```
!pip install -q git+https://github.com/pyro-ppl/numpyro.git
!pip install -Uq jax jaxlib matplotlib
```
to the top of [your tutorial](https://colab.sandbox.google.com/github/omarfsosa/numpyro/blob/tutorial%2Ftruncated/notebooks/source/truncated_distributions.ipynb)

Reply:
Awesome, thank you! I've just added this changes :)